src/lib/config.ts

```diff
@@ -0,0 +1,58 @@
+
+
+const TESTNET_CONFIG = {
+    VITE_PUBLIC_APP_NAME: 'sBTC Bridge Testnet',
+    VITE_PUBLIC_APP_VERSION: '1.0.0',
+    VITE_ORIGIN: 'https://sbtc.world',
+    VITE_NETWORK: 'testnet',
+    VITE_SBTC_WALLET: 'tb1q6zlpyrzvzjcrf6dlsctcrh9yl3dwfktsw0nclq',
+    VITE_SBTC_CONTRACT_ID: 'ST3N4AJFZZYC4BK99H53XP8KDGXFGQ2PRSPNET8TN.sky-blue-elephant',
+    VITE_BRIDGE_API: 'http://localhost:3010/bridge-api/v1',
+    VITE_STACKS_API: 'https://api.testnet.hiro.so',
+    VITE_STACKS_EXPLORER: 'https://explorer.hiro.so',
```

Comment:
https://explorer.hiro.so/?chain=testnet

Reply:
> https://explorer.hiro.so/?chain=testnet

we append this on check of CONFIG.network.